pkg/volume/util/resize_util_test.go

```diff
@@ -196,5 +278,24 @@ func getPVC(conditions []v1.PersistentVolumeClaimCondition) *v1.PersistentVolume
 			},
 		},
 	}
-	return pvc
+	return pvcModifier{pvc}
+}
+
+func (m pvcModifier) modifyStorageResourceStatus(status v1.ClaimResourceStatus) pvcModifier {
```

Comment:
very low prio nit: `withStorageResourceStatus` and `withResourceStatus` below.

Reply:
fixed.